Tools/Pipeline/Windows/ContentIcons.cs

```diff
@@ -25,6 +25,7 @@ public class ContentIcons: IDisposable
         public ContentIcons()
         {
             Icons = new ImageList();
+            Icons.ColorDepth = ColorDepth.Depth32Bit;
```

Comment:
Ok, so now -without the ColorDepth fix- I see black background whether I build for x86 or x64!
This is getting very weird!

Setting the ColorDepth to 32bit always fix it, so no worries!